scene/gui/text_edit.cpp

```diff
@@ -427,14 +428,18 @@ void TextEdit::Text::remove_range(int p_from_line, int p_to_line) {
 		if (text_line.width == max_line_width) {
 			max_line_width_dirty = true;
 		}
-		total_visible_line_count -= text_line.line_count;
+		if (!text_line.hidden) {
+			total_visible_line_count -= text_line.line_count;
+		}
```

Comment:
Doing it like it's done in https://github.com/godotengine/godot/pull/97898 seems better, since there's no need to update max w/h if a hidden line is removed.

Reply:
You are right. I changed it now.